reana_client/cli/workflow.py

```diff
@@ -111,6 +111,20 @@ def workflow_execution_group(ctx):
     "Use --filter `<columm_name>=<column_value>` pairs. "
     "Available filters are `name` and `status`.",
 )
+@click.option(
+    "--include-progress",
+    "include_progress",
+    is_flag=True,
+    default=False,
+    help="Include progress information of the workflows.",
+)
+@click.option(
+    "--include-workspace-size",
+    "include_workspace_size",
+    is_flag=True,
+    default=False,
+    help="Include size information of the workspace.",
+)
```

Comment:
I think we can set the defaults to `None` here, so we avoid sending these as query params all the time.